ios/sdk/src/JitsiMeet.m

```diff
@@ -50,7 +54,7 @@ + (instancetype)sharedInstance {
 
 - (instancetype)init {
     if (self = [super init]) {
-        // Initialize the on and only bridge for interfacing with React Native.
+        // Initialize the one and only bridge for interfacing with React Native.
         _bridgeWrapper = [[RCTBridgeWrapper alloc] init];
```

Comment:
We should only create the bridge if it doesn't exist yet, since an application could have instantiated it early now.

Reply:
since JitsiMeet is a singleton will be called only once